cns/requestcontroller/kubecontroller/apiclientinterface.go

```diff
@@ -0,0 +1,18 @@
+package kubecontroller
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ObjectKey identifies a Kubernetes Object.
+type ObjectKey = types.NamespacedName
```

Comment:
I don't think this is used anywhere, it can be deleted

Reply:
You're right, since checks take so long though, merging this PR and will remove in next iteration